examples/ng_networking/README.md

```diff
@@ -0,0 +1,59 @@
+# ng_networking example
+
+## Connecting RIOT native and the Linux host
+
+To establish a connection between RIOT and the Linux host, the following dependencies are required:
+
+    sudo apt-get install bridge-utils openvpn
+
+After installing, create a tap interface (to which RIOT will connect) and a bridge (to which Linux will connect):
+
+    sudo brctl addbr br0
+    sudo ip link set br0 up
+    sudo openvpn --mktun --dev tap0
+    sudo ip link set tap0 up
+    sudo brctl addif br0 tap0
```

Comment:
Here you can use `iproute2`, too. But why do you propose to create a bridge anyways?